docs/CDCHBase-sparksink.md

```diff
@@ -1,16 +1,24 @@
-CDC HBase Sink
-==========
+# CDC HBase Sink
 
-CDAP Plugin for Change Data Capture (CDC) in HBase using the Spark Framework. Plugin can be configured only for real-time pipelines.
+Description
+-----------
+CDAP Realtime Plugin for Change Data Capture (CDC) in HBase.
```

Comment:
This plugin takes input from a CDC source and writes the changes to HBase. It will write to the HBase instance running on the cluster.

Reply:
fixed